ros/src/tl_detector/tl_detector.py

```diff
@@ -265,6 +281,8 @@ def traffic_cb(self, msg):
         Finds next closest traffic light in front, takes its ground truth state
         and publishes to /traffic_waypoint
         """
+        self.tf_tst = msg.lights[0].state
+        return
```

Comment:
Is this return here right?

Reply:
i think it wont hurt to have a return here but could be removed as it brings no value.